framework/include/media/SocketOutputDataSource.h

```diff
@@ -0,0 +1,135 @@
+/* ****************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+/**
+ * @ingroup MEDIA
+ * @{
+ */
+
+/**
+ * @file media/SocketOutputDataSource.h
+ * @brief Media SocketOutputDataSource APIs
+ */
+
+#ifndef __MEDIA_SOCKETOUTPUTDATASOURCE_H
+#define __MEDIA_SOCKETOUTPUTDATASOURCE_H
+
+#ifndef INVALID_SOCKET
```

Comment:
Currently there's no return value using this, also app doesn't access this.
Then we'd better move this to cpp file

Reply:
Fixed